components/SearchBar/SearchBar.tsx

```diff
@@ -42,20 +50,32 @@ const SearchBarWrapper = styled.div`
   }
 `;
 
+const Table = styled.table`
```

Comment:
added custom table tags now, so that any changes can be made in the future.  They were the location of debug styling.